frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -716,6 +270,49 @@ export const VmActionFactory = {
       }),
     accessReview: asAccessReview(kindObj, vm, 'delete'),
   }),
+  // disabled until https://issues.redhat.com/browse/CNV-9746 is implemented
```

Comment:
this code should be removed, when https://issues.redhat.com/browse/CNV-9746
we will not use this code